### PR TITLE
Alignment and padding

### PR DESCRIPTION
**Pull request by realedk**
Added some alignment and padding. How does it look?
Screenshot from 2021-11-16 23-50-16

### CODE REVIEW

**Comment by RockSolid1106** [comment]
Great! How about centering the Submit button if the top stuff is centered too?
I just made the most basic configurator I could, cuz I'm no web designer, lol.

File: Extension/options.html [hunk 1]
```diff
@@ -63,6 +67,11 @@ <h3 style="padding: 0; margin: 0.3em;">Misc:</h3>
     font-family: 'Inter', sans-serif;
     font-size: 0.8em;
   }
+
+  #header {
+    text-align: left;
```
Comment (RockSolid1106) [inline Extension/options.html:72]: Did you upload some old code? This should be center and I've tested that.

Comment (EDKTech) [reply Extension/options.html:72]: No, I added a div for the header so the header would be centered and the content wouldn't.

**Comment by EDKTech** [comment]
> Great! How about centering the Submit button if the top stuff is centered too? I just made the most basic configurator I could, cuz I'm no web designer, lol.
> 
> Something like this: image

Yeah, I tried to do that and it ended up hanging to the left for some reason lol
Great job! It looks nice!

**Comment by EDKTech** [comment]
Oh yeah, I accidentally cut out the orange color

File: Extension/options.html (hunk 1 above)
Comment (RockSolid1106) [reply Extension/options.html:72]: No, I'm talking about the text-align: left..
Shouldn't it be center? I tested that and it left aligned the header.

**Comment by RockSolid1106** [comment]
> Yeah, I tried to do that and it ended up hanging to the left for some reason lol
Happened to me as well, seems there is some problem with the margins. You have to set the position absolute, then left to 40% to center it. Weird.


> Oh yeah, I accidentally cut out the orange color
Will add it then.

**Pull request closed without merging**